src/api/app/controllers/webui/package_controller.rb

```diff
@@ -701,9 +701,10 @@ def save_file
           errors << 'No file or URI given'
         end
       end
-
-    rescue APIException, StandardError => exception
-      errors << exception.message
+    rescue ActiveXML::Transport::Error => e
```

Comment:
Whether this qualifies as being ok to do I don't know, but there are a couple of places where we catch that exception. In some of them we show them as error in the ui.